src/components/icons/SVGWrapper/createSVGWrapper.tsx

```diff
@@ -0,0 +1,10 @@
+import { ReactNode } from "react";
+import { SVGWrapper, SVGWrapperProps } from "./SVGWrapper";
+
+export const createSVGWrapper = (node: ReactNode) => {
```

Comment:
**question**: do we need this wrapper?

Can't we use `<SVGWrapper/>` as component?

Reply:
With createSVGWrapper we don't have to import types for props in each icon component